rails/app/controllers/admin/settings_controller.rb

```diff
@@ -79,97 +74,53 @@ def show
   end
 
   # GET /admin/settings/new
-  # GET /admin/settings/new.xml
   def new
-    # PUNDIT_REVIEW_AUTHORIZE
-    # PUNDIT_CHECK_AUTHORIZE
-    # authorize Admin::Setting
     @admin_settings = Admin::Settings.new
-    @scope = nil
-
-
-    respond_to do |format|
-      format.html # new.html.erb
-      format.xml  { render :xml => @admin_settings }
-    end
+    # renders new.html.haml
   end
 
   # GET /admin/settings/1/edit
   def edit
-    # PUNDIT_REVIEW_AUTHORIZE
-    # PUNDIT_CHECK_AUTHORIZE (did not find instance)
-    # authorize @setting
     @admin_settings = Admin::Settings.find(params[:id])
-
+    # NP: 2020-08-19 TODO: This is not working nut it doesn't seem critical
     # Pull in the current theme default home page content, if it isn't set in the settings.
     if @admin_settings.home_page_content.nil? || @admin_settings.home_page_content.empty?
       render_to_string :partial => "home/project_info"
 
       @admin_settings.home_page_content = view_context.instance_variable_get(:@content_for_project_info)
       view_context.instance_variable_set(:@content_for_project_info, nil)
     end
-
-    if request.xhr?
-      render :partial => 'remote_form', :locals => { :admin_settings => @admin_settings }
-    end
+    # renders edit.html.haml
   end
 
   # POST /admin/settings
-  # POST /admin/settings.xml
   def create
-    # PUNDIT_REVIEW_AUTHORIZE
-    # PUNDIT_CHECK_AUTHORIZE
-    # authorize Admin::Setting
     @admin_settings = Admin::Settings.new(params[:admin_settings])
-    respond_to do |format|
-      if @admin_settings.save
-        flash[:notice] = 'Admin::Settings was successfully created.'
-        format.html { redirect_to(@admin_settings) }
-        format.xml  { render :xml => @admin_settings, :status => :created, :location => @admin_settings }
-      else
-        format.html { redirect_to(new_admin_setting_url) }
-        format.xml  { render :xml => @admin_settings.errors, :status => :unprocessable_entity }
-      end
+    if @admin_settings.save
+      flash[:notice] = 'Admin::Settings was successfully created.'
+      redirect_to @admin_settings
+    else
+      redirect_to new_admin_setting_url
```

Comment:
In the other controllers that I've been looking at from Ethan. 
this is doing
`render action: 'new'` or something like that instead of a redirect.
The render might be better than a redirect because I think it might keep any settings that were attempted to be set. Still visible in the form while, redirect will probably clear them.